### PR TITLE
New features modules as per issue #61

### LINKED ISSUE #61: Maintenance features - Database health - Row sizes as per Kentico recommendations.
Hi,

https://devnet.kentico.com/articles/maintenance-in-kentico?feed=Kentico

There are some elements of this article that aren't represented in KInspector, I'm not sure if you would prefer an idea per request?

It looks like a number of them are easy to implement - I'd be happy to look at them if you agree:

"Database Health"

1) Event Logs - Kentico recommend that the Event Log is set to 5,000-10,000 entries - it would be easy to query the table to check the size and return a warning if it's outside this range (and maybe critical for extreme settings?)

2) Web Farm Tasks - We've seen the table get quite large when a server is out of the web farm, I'm not sure of what would be considered 'too large' but if this table has a high number of rows (to be defined by Kentico) it most likely indicates that there is a problem.

### PR DESCRIPTION
**Pull request by martbrow**
Fixes #61

### CODE REVIEW

**Comment by petrsvihlik** [comment]
thanks for the commit! i did a review of the code and found a few little issues:

#### EventLogSize
- suffers from the same issue as https://github.com/Kentico/KInspector/issues/17 - the CMSLogSize is site-specific so you can't use `ExecuteAndGetScalar`. the setting should be checked for each site.
- it's good that you attach `eventLogSize` to `ModuleResults` for the `Status.Good`. why not to attach it to the warning as well? (btw the `ToString()` call is redundant...it's inferred from the usage)
- "Kentico recommend" should be "Kentico recommends"

#### OldWebFarmTasks
- "Please check the health of the web far" -> typo
- otherwise it looks good!

**Comment by petrsvihlik** [comment]
@martbrow are you planning on working on this issue? the pull request requires only minor adjustments for it to be accepted. thank you

**Comment by martbrow** [comment]
Hi, I do plan to fix this - I need to setup a multi site development environment first to be able to test it before submitting it.

**Comment by petrsvihlik** [comment]
:+1: great...just checking ;)

**Comment by ondrejsevcik** [comment]
@martbrow is this still alive?

**Comment by martbrow** [comment]
Hi,

It's been a while - but my commit seems to have created an issue?

Thanks.

**Comment by petrsvihlik** [comment]
Hi Martin,
the error says that

> appveyor was unable to build non-mergeable pull request

In other words, your branch (`martbrow:master`) has some changes that conflict with changes that has already been merged into `kentico:master`. To fix this you can either merge the latest version of `kentico:master` into your branch or rebase your branch on tip of the `master` branch.

Follow the steps here:
http://stackoverflow.com/questions/3876977/update-git-branches-from-master
And read the detailed description of the two approaches here:
https://www.atlassian.com/git/tutorials/merging-vs-rebasing/the-golden-rule-of-rebasing

**Comment by martbrow** [comment]
@petrsvihlik Hopefully ready to be accepted.

File: KInspector.Modules/Modules/EventLog/EventLogSizeModule.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    new Version("9.0")
+                },
+                Category = "Event log"
+            };
+        }
+
+
+        public ModuleResults GetResults(IInstanceInfo instanceInfo)
+        {
+            List<string> responses = new List<string>();
```
Comment (petrsvihlik) [inline KInspector.Modules/Modules/EventLog/EventLogSizeModule.cs]: This line can be removed

File: KInspector.Modules/Modules/EventLog/EventLogSizeModule.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                Status = Status.Good
+            };
+
+        }
+
+        private bool EventLogIsRecommendedSize(DataRow settingRow)
+        {
+            var size = Convert.ToInt32(settingRow["KeyValue"]);
+
+            if (size < 5000 || size > 10000)
```
Comment (petrsvihlik) [inline KInspector.Modules/Modules/EventLog/EventLogSizeModule.cs]: can be rewritten to a single-line return statement

**Comment by petrsvihlik** [comment]
Hi @martbrow , if you fix the two little remarks above, I'll accept the PR right away. Thanks!

**Comment by petrsvihlik** [comment]
Great job, @martbrow !

**Comment by martbrow** [comment]
Finally ;-)

**Pull request merged**